stored_requests/cache/cacher/cacher.go

```diff
@@ -0,0 +1,74 @@
+package cacher
+
+import (
+	"errors"
+)
+
+// Cacher is the cacher interface for the db_fetchers
+type Cacher interface {
```

Comment:
Wouldn’t we need Ping and Close for non-inmemory caches like Redis and Aerospike? Agreed we don’t need for inmemory 

Configure and Name are present because the code in PBS is unaware of what cache adapter we are using due to the blank import  https://github.com/prebid/prebid-server/blob/fa883640251577e166fc2a8aa1685dfe90579caf/pbs_light.go#L59

Reply:
You might be right... feel free to keep `Ping` and`Close` if you're calling them by the time the feature is complete.

It may also be that you're really thinking of _two_ interfaces, though. `Close()` is something we'd only want to call when Prebid Server is about to shut down. `Ping()` sounds like something we'd call once on startup, so PBS can fail fast if the connection is broken. Those aren't really things that the meat of the code generally needs to worry about.

There's definitely some art to this... but the Interface subsection of [this blog post](https://dave.cheney.net/2016/08/20/solid-go-design) made some reasonable points on the matter, so they're forming my underlying philosophy for now. Every principle has its exceptions, though, so we'll see how things look as the PR develops.

Reply:
``Ping()`` - should we eventually include that in the health check?

Reply:
ah! Yes, absolutely. That's sounds like a great idea.

Reply:
> It may also be that you're really thinking of two interfaces.

I like where this is headed. We could have one interface that deals with the connection to the cache database and another interface for the namespace/layer? So there would be one interface that handles the Redis connection and another interface that handles the DB and TTL settings for the Redis connection?

Reply:
yeah. my thoughts exactly ^^